src/blog/2024/05/why-you-need-a-low-code-platform.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Why Do You Need a Low-Code Platform?
```

Comment:
how about "Why you need a low-code platform" so that its less of a question and more of a fact/statement